docs/authenticate.js

```diff
@@ -312,47 +313,42 @@ const allReadBinding = Object.freeze({
  * some background on IAM roles.
  */
 function makeScoreAssetsWorldReadable() {
-  // TODO(janakr): Switch to using listEeAssets once #368 is in.
   // TODO(janakr): Consider sharing cache with list_ee_assets.js. Not trivial
   //  because that code does additional EE requests to look at geometries, so
   //  we would need a two-level cache, one raw and one with geometries.
-  ee.data.listAssets(eeLegacyPathPrefix + getDisaster(), {}, () => {})
-      .then((listResult) => {
-        if (!listResult) {
-          return;
-        }
-        const paths = new Set([getScoreAssetPath(), getBackupScoreAssetPath()]);
-        const numAssets = paths.size;
-        let foundAssets = 0;
-        for (const {id} of listResult.assets) {
-          if (paths.has(id)) {
-            foundAssets++;
-            ee.data.getIamPolicy(eeLegacyPrefix + id, () => {})
-                .then((policy) => {
-                  for (const binding of policy.bindings) {
-                    // Only want to modify 'reader' permissions.
-                    if (binding.role === 'roles/viewer') {
-                      if (!binding.members.includes('allUsers')) {
-                        binding.members.push('allUsers');
-                        ee.data.setIamPolicy(
-                            eeLegacyPrefix + id, policy, () => {});
-                      }
-                      return;
-                    }
-                  }
-                  // TODO(janakr): Do better. See what EE says.
-                  // If we got here, no roles/viewer binding. Use some
-                  // Javascript magic.
-                  const BindingConstructor = policy.bindings[0].constructor;
-                  policy.bindings.push(new BindingConstructor(allReadBinding));
-                  ee.data.setIamPolicy(eeLegacyPrefix + id, policy, () => {});
-                });
-          }
-          if (foundAssets === numAssets) {
-            return;
+  listEeAssets(eeLegacyPathPrefix + getDisaster()).then((listedAssets) => {
```

Comment:
I think the listEeAssets doc is wrong now, sorry I didn't catch it before. The return type is much simpler right?

Reply:
Sorry, what's wrong about it? It's a promise with an array of <id, type>, no?

Reply:
Ah, I guess we just don't do any actual processing of the type stuff until later so while the result does have those fields, we're not actually using them in listEeAssets so sort of strange to document them in the return type. but maybe not!